Closr/Restaurant/View/RestaurantTableViewCell.swift

```diff
@@ -1,24 +1,270 @@
 //
 //  RestaurantTableViewCell.swift
-//  Closr
+//  TableViewTest
 //
-//  Created by Zhitao on 2017-02-17.
-//  Copyright © 2017 closr. All rights reserved.
+//  Created by Yale的Mac on 2017/2/28.
+//  Copyright © 2017年 YaleMac. All rights reserved.
 //
 
 import UIKit
+import EasyPeasy
 
-class RestaurantTableViewCell: UITableViewCell {
+struct restaurantInformation {
+    
+    
+    var restaurantName=String()
+    var restaurantAddress=String()
+    var restaurantDist=String()
+    var restaurantPrice=String()
+    var restaurantCategory=String()
+    
+    
+    
+}
 
+class RestaurantTableViewCell: UITableViewCell {
     override func awakeFromNib() {
         super.awakeFromNib()
         // Initialization code
     }
-
+    
     override func setSelected(_ selected: Bool, animated: Bool) {
         super.setSelected(selected, animated: animated)
-
+        
         // Configure the view for the selected state
     }
-
+    
+    
+    override init(style: UITableViewCellStyle,reuseIdentifier reuseIdentifire: String?){
+        
+        super.init(style: style, reuseIdentifier: reuseIdentifire)
+        
+        
+        
+        
+        setUpViews()
+        
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    
+    
+    func setUpViews(){
+        
+        
+        
+        var subFrame = CGRect()
+        
+        
+        /*
+         subFrame.origin.x = CGFloat(RestaurantFontSpace.restaurantImageViewSpace.originX)
+         subFrame.origin.y = CGFloat(RestaurantFontSpace.restaurantImageViewSpace.originY)
+         subFrame.size.width = CGFloat(RestaurantFontSpace.restaurantImageViewSpace.sizeWidth)
+         subFrame.size.height = CGFloat(RestaurantFontSpace.restaurantImageViewSpace.sizeHeight)
+         
+         */
+        let  restaurantImageView = UIImageView()
```

Comment:
those subviews can be "lazy var", please take a look at other controllers/views for reference